Orange/data/sql/backend/postgres.py

```diff
@@ -22,6 +22,7 @@ class Psycopg2Backend(Backend):
     display_name = "PostgreSQL"
     connection_pool = None
     auto_create_extensions = True
+    missing_extension = ""
```

Comment:
This should probably be an instance attribute declared in `__init__` as it depends on the connection_params